### PR TITLE
Safe browser seo page

### PR DESCRIPTION
**Pull request by SekhmetDesign**
## Description
Adds a Safe Browser SEO page

## Issue / Bugzilla link
#6793 

## Testing
https://www-demo5.allizom.org/firefox/features/safebrowser/

### CODE REVIEW

File: media/static-bundles.json [hunk 1]
```diff
@@ -570,12 +570,6 @@
       ],
       "name": "firefox_whatsnew_67"
     },
-    {
-      "files": [
-        "css/firefox/whatsnew/whatsnew-67.0.5.scss"
-      ],
-      "name": "firefox_whatsnew_67.0.5"
```
Comment (craigcook) [inline media/static-bundles.json]: Looks like this was accidentally deleted. If you rebase from the current master branch it should restore it, though might cause some merge conflicts you'll need to resolve manually.

File: media/static-bundles.json [hunk 2]
```diff
@@ -1031,17 +1031,6 @@
...
-    {
-      "files": [
-        "js/base/uitour-lib.js",
-        "js/base/mozilla-fxa.js",
-        "js/base/mozilla-fxa-init.js",
-        "js/base/mozilla-fxa-form.js",
-        "js/base/mozilla-monitor-button.js",
-        "protocol/js/protocol-modal.js"
-      ],
-      "name": "firefox_whatsnew_67.0.5"
```
Comment (craigcook) [inline media/static-bundles.json]: This was also accidentally removed but a rebase should put it back.

**Comment by craigcook** [comment]
I've pushed this to a demo server at https://www-demo5.allizom.org/firefox/features/safebrowser/

File: media/static-bundles.json (hunk 2 above)
Comment (SekhmetDesign) [reply media/static-bundles.json]: this should be fixed now

File: media/static-bundles.json (hunk 1 above)
Comment (SekhmetDesign) [reply media/static-bundles.json]: this should be fixed now

**Comment by ejregithub** [comment]
@SekhmetDesign - QA is still in progress. I'll wait til we have all the comment/edits in, likely EOD tomorrow (Wed 5/29) and submit them to you then.  Only copy would change, no design/layout.

**Comment by amychurchwell** [comment]
@ejregithub I have some feedback about the copy but I saw we are expecting some changes. Is this PR the place to provide that feedback or should I comment on the copy doc?

File: bedrock/firefox/templates/firefox/features/safebrowser.html [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+        Cleaning up your on line space isn't just good feng shui, it can lessen the worry of being tracked. Separate
+        identities
+        for your work websites or your personal interests keep trackers confined to one area. Ads that have nothing to
+        do with
+        work won't pop-up in the middle of a presentation. For this purpose, we created Containers.
+        {% endtrans %}
+      </p>
+      <p>
+        {% trans %}
+        Containers help you stay organized and keep trackers in their place. Checking flight prices? Log onto the same
```
Comment (amychurchwell) [inline bedrock/firefox/templates/firefox/features/safebrowser.html]: Looks like you are missing links from the page copy. [The copy doc](https://docs.google.com/document/d/1C1n0x6YUXgkub07_EJiSB86u1PRQQU0mi_DzgRnT7zY/) shows that 'Containers' should link to https://addons.mozilla.org/firefox/addon/multi-account-containers/

We can do this in the following way:
```
{% trans url='https://addons.mozilla.org/firefox/addon/multi-account-containers/' %}
<a href="{{ url }}">Containers</a>...
```
`url` can be whatever variable name you would like. 🙂

File: bedrock/firefox/templates/firefox/features/safebrowser.html [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+        website
+        on different containers to make sure the price doesn't mysteriously go up the second time you visit.
+        {% endtrans %}
+      </p>
+      <p>
+        {% trans %}
+        If you're concerned about how much Facebook already knows about you, try the Facebook Container. It makes it
```
Comment (amychurchwell) [inline bedrock/firefox/templates/firefox/features/safebrowser.html]: 'Facebook Container' should link to https://blog.mozilla.org/firefox/facebook-container-extension/

File: bedrock/firefox/templates/firefox/features/safebrowser.html [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+        A password manager should help you create strong passwords and protect them across accounts. Most people can't
+        remember
+        every single password, especially if they're secure passwords. (Unless you have a photographic memory, in which
+        case,
+        congratulations!)
+        {% endtrans %}
+      </p>
+      <p>
+        {% trans %}
+        Firefox Password Manager stores all usernames and passwords in an encrypted format on your hard drive. We don't
```
Comment (amychurchwell) [inline bedrock/firefox/templates/firefox/features/safebrowser.html]: 'Firefox Password Manager' links to https://support.mozilla.org/kb/password-manager-remember-delete-change-and-import

'Master Password' links to https://support.mozilla.org/kb/use-master-password-protect-stored-logins

File: bedrock/firefox/templates/firefox/features/safebrowser.html (hunk 3 above)
Comment (amychurchwell) [reply bedrock/firefox/templates/firefox/features/safebrowser.html]: Keep in mind that we do not hard code locale in links (ex: en-US) so the user's locale is loaded.

File: bedrock/firefox/templates/firefox/features/safebrowser.html [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+    <div class="download-firefox">
+      {{ download_firefox(dom_id='safebrowser-hero-download', download_location='primary cta') }}
+    </div>
+  </div>
+</div>
+<div class="mzp-l-content">
+  <article class="mzp-c-article">
+    <p class="mzp-c-article-intro">
+      {% trans %}
+      Building a safe browser in an art and a science because there are no set rules. At Firefox, we believe you have
```
Comment (amychurchwell) [inline bedrock/firefox/templates/firefox/features/safebrowser.html]: I think this should read:
> Building a safe browse is an art and a science...

File: bedrock/firefox/templates/firefox/features/safebrowser.html [hunk 7]
```diff
@@ -0,0 +1,222 @@
...
+      the right to privacy on the internet. After all, the web is your home away from home, your workspace or your
+      storage unit.
+      {% endtrans %}
+    </p>
+    <img src="{{ static('img/firefox/features/safebrowser-lock.png')}}" alt="">
+    <p>
+      {% trans %}
+      When stepping inside a home or office, it's your right to be comfortable, safe and secure. There's no looking
```
Comment (amychurchwell) [inline bedrock/firefox/templates/firefox/features/safebrowser.html]: Should this be:

> There's no one looking over your shoulder

?

File: bedrock/firefox/templates/firefox/features/safebrowser.html [hunk 8]
```diff
@@ -0,0 +1,222 @@
...
+      over your shoulder. You can shut the blinds or choose not to answer the door. The web should be no different.
+      {% endtrans %}
+    </p>
+    <p>
+      {% trans %}
+      That is why we make Firefox with security features and add-ons that help us make the web safer for you. In this
```
Comment (amychurchwell) [inline bedrock/firefox/templates/firefox/features/safebrowser.html]: I think this should read:

> At this tech company...

File: bedrock/firefox/templates/firefox/features/safebrowser.html [hunk 9]
```diff
@@ -0,0 +1,222 @@
...
+        giving away less about what you do and where you go online.
+        {% endtrans %}
+      </p>
+    </section>
+
+    <section class="article-section">
+      <h2>{{ _('Containers') }}</h2>
+      <p>
+        {% trans %}
+        Cleaning up your on line space isn't just good feng shui, it can lessen the worry of being tracked. Separate
```
Comment (amychurchwell) [inline bedrock/firefox/templates/firefox/features/safebrowser.html]: > Cleaning up your online space

File: bedrock/firefox/templates/firefox/features/safebrowser.html [hunk 10]
```diff
@@ -0,0 +1,222 @@
...
+
+    <section class="article-section">
+      <h2>{{ _('Firefox Monitor') }}</h2>
+      <p>
+        {% trans %}
+        It's hard to go a week without hearing of a new data breach. If you're a human who lives and works, you probably
+        don't
+        have the time to keep track of data news. We thought it might be nice to have a tool that kept watch on accounts
+        and
+        could let you know if anything fishy is going on.
```
Comment (amychurchwell) [inline bedrock/firefox/templates/firefox/features/safebrowser.html]: We should leave an l10n note about the meaning of "fishy" in this context.
https://bedrock.readthedocs.io/en/latest/l10n.html#translating-with-lang-files
```
{# L10n: "Fishy" in this context is an adjective meaning suspicious or questionable. #}
```

**Comment by ejregithub** [comment]
@amychurchwell @SekhmetDesign  - let's hold on changes to copy as I think they will be significant. I would insert the links though.

@amychurchwell - those comments to localizers are super-helpful, thank you (I hope that copy doesn't change!)

**Comment by craigcook** [comment]
Just a minor thing but I noticed we don't seem to be using curly quotes or apostrophes in the text, which can probably be blamed on copy-paste. House style prefers curly punctuation (and everything is UTF8 so you don't need to escape the characters).

**Comment by amychurchwell** [comment]
@SekhmetDesign thank you so much for all your work on this, just wanted to let you know that I'll be taking over the PR 😄

**Comment by craigcook** [comment]
Closing in favor of https://github.com/mozilla/bedrock/pull/7288

**Pull request closed without merging**